src/Integration.TeamExplorer/TeamExplorerController.cs

```diff
@@ -21,40 +21,32 @@
 using System;
 using System.ComponentModel.Composition;
 using System.Diagnostics;
-using System.Globalization;
 using Microsoft.TeamFoundation.Controls;
-using Microsoft.VisualStudio.Shell;
-using SonarLint.VisualStudio.Integration.Resources;
+using SonarLint.VisualStudio.Infrastructure.VS;
 
 namespace SonarLint.VisualStudio.Integration.TeamExplorer
 {
     [Export(typeof(ITeamExplorerController))]
     [PartCreationPolicy(CreationPolicy.Shared)]
     public class TeamExplorerController : ITeamExplorerController
     {
-        private readonly ITeamExplorer teamExplorer;
-
-        internal /* testing purposes */ ITeamExplorer TeamExplorer => this.teamExplorer;
+        private readonly IVsUIServiceOperation vSServiceOperation;
 
         [ImportingConstructor]
-        public TeamExplorerController([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider)
+        public TeamExplorerController(IVsUIServiceOperation vSServiceOperation)
         {
-            if (serviceProvider == null)
```

Comment:
I removed this instead of checking if vsserviceOperation is null because we have a mef test to check that the import is successful